roles/adw/handlers/main.yml

```diff
@@ -1,13 +1,43 @@
 ---
 # handlers file for adw
-- name: validate nginx configuration
-  command: nginx -t -c /etc/nginx/nginx.conf
-  changed_when: false
-- name: reload nginx
-  service: name=nginx state=reloaded
+# - name: validate nginx configuration
```

Comment:
why did you leave this here?

Reply:
wasn't sure of the purpose of this handler and just commented it as it is not notified anywhere. Will remove it

Reply:
removed